docs/Cloud_integrations/LoRaWAN/LoRagateway.md

```diff
@@ -0,0 +1,101 @@
+
+1. Go to *Integration > LoRaWAN Gateways*.
+2. Click **Connect** in The Things Stack card.
+3. Provide the required data and click **Save**.
+
+!!! Important
+    The integration data is not validated when setting the configuration. It will be validated once you add the first LoRaWAN device. If the validation results in an error when adding the first LoRaWAN Gateway device, return to the integration panel and correct the configuration.
+
+## Adding LoRaWAN Gateway devices
+
+To add a LoRaWAN Gateway:
+
+1. Go to *Device Inventory*.
+2. Click **Add device** and then **Add device manually** under LORAWAN GATEWAY DEVICES.
+3. Select connection via the Bootstrap Server or via the Management Server.
+
+    !!! Info
+        Connecting devices via the Bootstrap Server is unavailable in the Developer plan.
+
+4. Follow the steps in the device creator.
+
+Adding a LoRaWAN Gateway via the Management Server consists of two steps: 
+
+- **Step 1: Configure your device connection**
+- **Step 2: Connect your device** 
+
+Adding a LoRaWAN Gateway via the Bootstrap Server consists of three steps: 
+
+- **Step 1: Configure your device connection** 
+- **Step 2: Configure server connection**
+- **Step 3: Connect your device**
+
+Below you can find the description of each step:
+
+### Configure your device connection
+		
+In this step provide the information about the LoRaWAN Gateway device and choose device connection settings.
+		
+In the section *Device credentials*, provide the following information:
+
+- *Gateway EUI* - choose one of two options:
+    - *Manufacturer* - select this option, if you have a gateway from a manufacturer with a Gateway EUI. The following field is available for this option:
+        - *Endpoint name - Gateway EUI*: A 64-bit extended unique identifier for your gateway. It should be provided by the manufacturer or printed on the gateway packaging. It will become the endpoint name by which the platform server will identify the device.
+    - *No gateway EUI* - select this option if your gateway doesn't have a Gateway EUI. The following field is available for this option:
+        - *Endpoint name*: Provide an endpoint name by which the platform server will identify your device.
+- *Gateway ID*:  Unique identifier for your gateway that will be used by The Thinks Stack to identify your device.
+- *Friendly name - Gateway name*: An optional human-readable name to help you identify your gateway. The Things Stack will use this name instead of the gateway ID to refer to your gateway.
+
+In the section *Settings*, select your Frequency plan. The frequency plan defines data rates that your end device or gateway is set up to use. The gateway and end devices within reach must use the same frequency plan to communicate. 
```

Comment:
```suggestion
In the section *Settings*, select your frequency plan. The frequency plan defines channels, data rates and radio configuration that your end device or gateway is set up to use. The gateway and end devices within reach must use the same frequency plan to communicate. 
```

I know that it's mostly copied from the Coiote, but `data rates` alone is not enough.